### ISSUE TITLE
Clicking on the name in the "Whois" starts a private chat

### ISSUE THREAD

**Issue by Runster** [hexchat/hexchat#1756]
Hello,

i have a small idea for the "_Whois_-Information":

Is it possible, to make the nickname in the _whois_ clickable?

So, if I'm requesting the _Whois_-information for a user, a click on his nickname (in the response of the requested information) opens a window for a private chat with this user.

Thanks for comments!

**Comment by TingPing** [comment]
The problem is just the default theme choosing poor characters. If you go to *Settings > Text Events* and modify any of the *Whois* events to not have *[* and *]* around the nick it will work.

**Comment by Runster** [comment]
I've changed the default theme to another one, but it is still not working. I also tried to remove the [ and ] around the nick, but that also doesn't help.


Maybe my problem was not clear.
For example: If you are requesting a Whois for a User one of the rows you get is:

_[User] is logged in as User_

I want to start a private chat (/query USER) to this user, if I'm clicking on his name..

Thanks for your help so far!

**Comment by Arnavion** [comment]
Yes, even when the `[]` right-clicking doesn't generate a context menu.

**Comment by TingPing** [comment]
Works as expected here...

**Comment by Arnavion** [comment]
That's true, but for me whois output goes to the network tab. I can't find which setting does that.

**Comment by TingPing** [comment]
`/set irc_whois_front`

**Comment by Runster** [comment]
Okay, I have to say that I am doing this request in the network tab - so there is no user. Still possible?

**Comment by TingPing** [comment]
No, the nick detection works by checking the word you hover over against users in the channel. In theory a server tab could look for every user in the network we know of but I don't think this is necessary.
